homeassistant/components/isy994/light.py

```diff
@@ -76,14 +80,21 @@ def turn_off(self, **kwargs) -> None:
     def on_update(self, event: object) -> None:
         """Save brightness in the update event from the ISY994 Node."""
         if self._node.status not in (0, ISY_VALUE_UNKNOWN):
-            self._last_brightness = self._node.status
+            self._last_brightness = (
+                self._node.status
+                if self._node.uom != UOM_PERCENTAGE
+                else int(self._node.status * 255.0 / 100.0)
```

Comment:
```suggestion
                else round(self._node.status * 255.0 / 100.0)
```

Reply:
If the ternary spawns multiple lines, the Home Assistant standard is to unpack it.